appengine/pond/duck/board/js/board.js

```diff
@@ -62,42 +74,56 @@ Pond.Duck.Board.createDuck = function() {
 };
 
 /**
- * Callback for when all the ducks for a user are fetched.s
+ * Callback for copy/delete action.
+ * @param {string} action Action attempted.
  */
-Pond.Duck.Board.getDucksCallback = function() {
-  var text;
-  if (this.status == 200) {
-    var duckList = JSON.parse(this.responseText);
-    Pond.Duck.Board.setTemplate(duckList)
-  } else {
-    text = BlocklyGames.getMsg('Games_httpRequestError') + '\nStatus: '
-        + this.status;
-  }
+Pond.Duck.Board.createGetDucksCallback = function(action) {
+  return function() {
+    var text;
+    if (this.status == 200) {
+      Pond.Duck.Board.refreshDuckList.call(this);
+      var meta = JSON.parse(this.responseText);
+      text = 'Duck ' + action + ' with key: ' + meta['duck_key'];
```

Comment:
Do we need to pop up a dialog if the result will be shown in the update list?

Reply:
Added for our sake, testing.